src/hooks/__tests__/useModalContext.test.tsx

```diff
@@ -58,24 +68,100 @@ it('should call setHasHeader when modal has header', () => {
     </ModalContextProvider>
   )
 
-  expect(mockSetHasHeader).toHaveBeenCalledTimes(1)
+  expect(mockSetPart).toHaveBeenCalledTimes(1)
+  expect(mockSetPart).toHaveBeenCalledWith('hasHeader', true)
 })
 
-it('should not call setHasHeader when modal has no header', () => {
-  const mockSetHasHeader = jest.fn()
+it('should call setPart when modal has left sidebar', () => {
```

Comment:
since these tests are about the components behavior, I think they should be located in the components tests. I don't think you need to test combinations of calls, since these are unit tests.

Reply:
done ✔️